docs/source/en/model_doc/vitpose.md

```diff
@@ -0,0 +1,250 @@
+person_model = RTDetrForObjectDetection.from_pretrained("PekingU/rtdetr_r50vd_coco_o365")
+inputs = person_image_processor(images=image, return_tensors="pt")
+
+with torch.no_grad():
+  outputs = person_model(**inputs)
+
+results = person_image_processor.post_process_object_detection(outputs, target_sizes=torch.tensor([image.size[::-1]]), threshold=0.3)
+
+def pascal_voc_to_coco(bboxes: np.ndarray) -> np.ndarray:
+    """
+    Converts bounding boxes from the Pascal VOC format to the COCO format.
+
+    In other words, converts from (top_left_x, top_left_y, bottom_right_x, bottom_right_y) format
+    to (top_left_x, top_left_y, width, height).
+
+    Args:
+        bboxes (`np.ndarray` of shape `(batch_size, 4)):
+            Bounding boxes in Pascal VOC format.
+
+    Returns:
+        `np.ndarray` of shape `(batch_size, 4) in COCO format.
+    """
+    bboxes[:, 2] = bboxes[:, 2] - bboxes[:, 0]
+    bboxes[:, 3] = bboxes[:, 3] - bboxes[:, 1]
+
+    return bboxes
+
+# 0 index indicates human label in COCO
+boxes = results[0]['boxes'][results[0]['labels'] == 0]
+boxes = [pascal_voc_to_coco(boxes.cpu().numpy())]
+
+image_processor = VitPoseImageProcessor.from_pretrained("nielsr/vitpose-base-simple")
+model = VitPoseForPoseEstimation.from_pretrained("nielsr/vitpose-base-simple")
+
+# Stage 2. Run ViTPose
+pixel_values = image_processor(image, boxes=boxes, return_tensors="pt").pixel_values
+
+with torch.no_grad():
+  outputs = model(pixel_values)
+
+pose_results = image_processor.post_process_pose_estimation(outputs, boxes=boxes)[0]
+
+for pose_result in pose_results:
+  for keypoint in pose_result['keypoints']:
+    x, y, score = keypoint
+    print(f"coordinate : [{x}, {y}], score : {score}")
+
+def visualize_keypoints(img,
+  pose_result,
+  skeleton=None,
+  kpt_score_thr=0.3,
+  pose_kpt_color=None,
+  pose_link_color=None,
+  radius=4,
+  thickness=1,
+  show_keypoint_weight=False):
+    """Draw keypoints and links on an image.
+
+    Args:
+            img (str or Tensor): The image to draw poses on. If an image array
+                is given, id will be modified in-place.
+            pose_result (list[kpts]): The poses to draw. Each element kpts is
+                a set of K keypoints as an Kx3 numpy.ndarray, where each
+                keypoint is represented as x, y, score.
+            kpt_score_thr (float, optional): Minimum score of keypoints
+                to be shown. Default: 0.3.
+            pose_kpt_color (np.array[Nx3]`): Color of N keypoints. If None,
+                the keypoint will not be drawn.
+            pose_link_color (np.array[Mx3]): Color of M links. If None, the
+                links will not be drawn.
+            thickness (int): Thickness of lines.
+    """
+    img = img.copy()
+    img_h, img_w, _ = img.shape
+
+    for kpts in pose_result:
+        kpts = np.array(kpts, copy=False)
+
+        # draw each point on image
```

Comment:
let's split up some of this functionality into smaller function i.e. this function calls `draw_points` and `draw_links`

Reply:
no problem